### PR TITLE
docs: public preview callout css updates

### PR DESCRIPTION
**Pull request by kay-kim**
Increasing contrast for preview in dark mode + update where we send people when they contact us.

### CODE REVIEW

File: doc/user/layouts/shortcodes/private-preview.html [hunk 1]
```diff
@@ -6,8 +6,8 @@
   It isn't subject to our backwards compatibility guarantees.
   {{if not (.Get "enabled-by-default")}}
   <br><br>
-  You must
-  <a href="https://materialize.com/contact/">contact us</a>
-  to enable this feature in your Materialize region.
+  To enable this feature in your Materialize region,
+  <a href="https://materialize.com/docs/support/">contact your Materialize
```
Comment (arusahni) [inline doc/user/layouts/shortcodes/private-preview.html:10]: The "your" here is orphaned. I think we should either say "contact Materialize support" or "contact your Materialize support contact."

The former feels preferable?

**Review by arusahni — APPROVED** [review]
Looks much better, thank you!

File: doc/user/layouts/shortcodes/private-preview.html (hunk 1 above)
Comment (morsapaes) [reply doc/user/layouts/shortcodes/private-preview.html:10]: Nice spot, Aru! We should be consistent with the rest of the documentation, where we...mostly use "contact our team".

Comment (kay-kim) [reply doc/user/layouts/shortcodes/private-preview.html:10]: Definitely happy to converge but just making sure that we're not erasing a distinction where in some cases, we want our customers to contact us per their usual support channel and other cases, where we want people to contact us to sign up to schedule a demo or guided trial or whatnots.  

Basically, is it: a) maintain a distinction between general reachouts vs support through the wording or b) just go with the same wording but rely on the specific links to do the differentiating once they click. 

If this hasn't been an issue, can definitely go with "contact our team" if that's been what we've mostly been using.

Comment (morsapaes) [reply doc/user/layouts/shortcodes/private-preview.html:10]: At least the ones I'm referring to are used in the same context (pointing to `/support`).

File: doc/user/layouts/shortcodes/private-preview.html [hunk 2]
```diff
@@ -6,8 +6,8 @@
...
+  support</a>.
```
Comment (morsapaes) [inline doc/user/layouts/shortcodes/private-preview.html:11]:
```suggestion
  <a href="https://materialize.com/docs/support/">contact our team</a>.
```

**Review by morsapaes — APPROVED** [review]

File: doc/user/layouts/shortcodes/private-preview.html (hunk 2 above)
Comment (kay-kim) [reply doc/user/layouts/shortcodes/private-preview.html]: Thanks much!

**Pull request merged**